app/templates/settings/profile.hbs

```diff
@@ -8,4 +8,6 @@
   <div class="settings">
     {{user-settings-form user=model}}
   </div>
+
+  {{github-connect state=state}}
```

Comment:
You don't actually need to specify a state, no?

Also, this does not look good here. Do you wanna remove it before merge and deal with placing it in a separate PR for issue #1267 ?

I can add a PR for it.